### ISSUE TITLE
TypeError: Tracy\Dumper\Describer::addPropertyTo(): Argument #2 ($k) must be of type string, int given

### ISSUE THREAD

**Issue by Gappa** [nette/tracy#559]
Version: 2.10.1

### Bug Description

![image](https://user-images.githubusercontent.com/749981/228456891-45422db1-bfb5-4270-bd9a-92d9b545e3c3.png)

``` 
TypeError: Tracy\Dumper\Describer::addPropertyTo(): Argument #2 ($k) must be of type string, int given, called in
/{path_to_project}/vendor/tracy/tracy/src/Tracy/Dumper/Exposer.php on line 27 and defined in
/{path_to_project}/vendor/tracy/tracy/src/Tracy/Dumper/Describer.php:235
``` 

### Steps To Reproduce

In a latte template:

``` latte
{foreach new \ArrayIterator(['a', 'b', 'c']) as $foo}
	{$foo}
{/foreach}
``` 

### Expected Behavior

No error is shown in the Tracy bar. 
Previous versions did not have this problem.

### Possible Solution

Retyping the `$k` parameter to `string`:

- https://github.com/nette/tracy/blob/744ab824e25cbf027fc820346ad0713ed8a961a6/src/Tracy/Dumper/Exposer.php#L25-L27

As it is already done here:

- https://github.com/nette/tracy/blob/744ab824e25cbf027fc820346ad0713ed8a961a6/src/Tracy/Dumper/Exposer.php#L29-L37

**Comment by dg** [comment]
I can't replicate the bug. I would rather say that it is caused by some parameter added to `$this->template`.

**Comment by Gappa** [comment]
Doh, you're right, I fiddled with it so much that I mucked up the test case.

So, the correct steps:

1. Create empty nette/web-project.
2. Enable debug mode.
3. Modify HomePresenter (add the actionDefault method):

``` php
final class HomePresenter extends Nette\Application\UI\Presenter
{
	public function actionDefault(): void
	{
		$this->template->foo = new \ArrayIterator(['a', 'b', 'c']);
	}
}
``` 

![image](https://user-images.githubusercontent.com/749981/228497548-31983271-d5bd-46da-ba81-17d5e1ace5bd.png)

But it should fail in any presenter, I just wanted to make it as clean as possible. 

The DIC panel also fails (It's disabled in my project) on the same error.
